playback/src/config.rs

```diff
@@ -147,12 +373,17 @@ impl Default for PlayerConfig {
             bitrate: Bitrate::default(),
             gapless: true,
             normalisation: false,
+            interpolation_quality: InterpolationQuality::default(),
+            sample_rate: SampleRate::default(),
             normalisation_type: NormalisationType::default(),
             normalisation_method: NormalisationMethod::default(),
             normalisation_pregain_db: 0.0,
             normalisation_threshold_dbfs: -2.0,
-            normalisation_attack_cf: duration_to_coefficient(Duration::from_millis(5)),
-            normalisation_release_cf: duration_to_coefficient(Duration::from_millis(100)),
+            // Dummy value. We can't use the default because
+            // no matter what it's dependent on the sample rate.
+            normalisation_attack_cf: 0.0,
+            // Same with release.
+            normalisation_release_cf: 0.0,
```

Comment:
They are assigned in main. The dummy values are never actually ever used as the real values are completely dependent on the sample rate. You could use sample_rate::default().duration_to_normalisation_coefficient and friend but it would still just be a dummy value, just a more expensive one.

Maybe a warning in the normaliser if it's created with the dummy values?

Reply:
How's this?

https://github.com/librespot-org/librespot/pull/1180/commits/c29ec3f4731d030fd321bd3defbaa752745886ad